keploy.sh

```diff
@@ -136,20 +136,31 @@ installKeploy (){
         fi
     }
 
+    cleanup_tmp() {
+        # Remove extracted files /tmp directory
+        tmp_files=("LICENSE" "README.md" "READMEes-Es.md" "README-UnitGen.md")
+        for file in "${tmp_files[@]}"; do
+            sudo rm -rf "/tmp/$file"
```

Comment:
You can also add a check to see if the file exists instead of directly removing them. Rest looks good.

Reply:
made the chage by adding `if [ -f "/tmp/$file" ];`